### PR TITLE
Change brand name from “React Storybook” to “Storybook”

### PR DESCRIPTION
**Pull request by ndelangen**
## What I did
- I moved and renamed storybooks
- changed wording from "React Storybook" to "Storybook"

### CODE REVIEW

File: addons/storyshots/README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ To use StoryShots, you must use your existing Storybook stories as the input for
 Add the following module into your app.
 
 ```sh
-npm install -save-dev storyshots
+npm install -save-dev @storybook/addon-storyshots
```
Comment (shilman) [inline addons/storyshots/README.md]: should be `--save-dev`?

**Review by shilman — CHANGES_REQUESTED** [review]
Minor change unrelated to your change, but on the same line.

File: addons/storyshots/README.md (hunk 1 above)
Comment (ndelangen) [inline addons/storyshots/README.md]: Nice catch, can you approve now?

**Comment by UsulPro** [comment]
So we officialy rename `React Storybook` to `Storybook`?

seems logical 👍 
the only doubt how will this affect the indexing in web search?
compare [react storybook](https://www.google.ru/search?ie=UTF-8&hl=ru&q=react%20storybook&gws_rd=ssl) vs [storybook](https://www.google.ru/search?ie=UTF-8&hl=ru&q=storybook&gws_rd=ssl)

**Comment by ndelangen** [comment]
It's still the top hit? I see no problem with this.

It's a needed step.

**Pull request merged**